### PR TITLE
Abstract and canonical type for Lighting

### PR DESCRIPTION
**Pull request by RathishKJ**

### CODE REVIEW

File: ontology/yaml/resources/LIGHTING/entity_types/LCM.yaml [hunk 1]
```diff
@@ -21,3 +21,52 @@ LCM_SS:
...
+  - IM
+
+LCM_LSC:
+  description: "Standard lighting with switch control."
+  is_canonical: true
+  implements:
+  - LCM
+  - LSC
+
+LCM_PWM:
```
Comment (tasodorff) [inline ontology/yaml/resources/LIGHTING/entity_types/LCM.yaml:46]: do we not have any other data for the device? if this is a control module shouldnt there be control?

Comment (RathishKJ) [reply ontology/yaml/resources/LIGHTING/entity_types/LCM.yaml]: The single canonical has been deleted and the unified canonical LGRP_OS_IM_LSC_LC_PWM_SCZ has been created.

File: ontology/yaml/resources/LIGHTING/entity_types/LCM.yaml [hunk 2]
```diff
@@ -21,3 +21,4 @@ LCM_SS:
   implements:
   - LCM
   - SS
+
```
Comment (tasodorff) [inline ontology/yaml/resources/LIGHTING/entity_types/LCM.yaml:24]: revert if just whitespace.

**Review by tasodorff — CHANGES_REQUESTED** [review]
remove whitespace

File: ontology/yaml/resources/LIGHTING/entity_types/LCM.yaml (hunk 2 above)
Comment (RathishKJ) [reply ontology/yaml/resources/LIGHTING/entity_types/LCM.yaml]: Yes, It is a white space and has been removed...

**Review by tasodorff — APPROVED** [review]

**Pull request merged**